tensorflow_datasets/image/cityscapes.py

```diff
@@ -194,24 +207,24 @@ def calculate_num_shards(split_size, features_size_mb):
       return math.ceil(split_size / instances_per_shard)
 
     splits = [
-      tfds.core.SplitGenerator(
-          name=tfds.Split.TRAIN,
-          num_shards=calculate_num_shards(2975, features_size_mb),
-          gen_kwargs={
-              feat:os.path.join(path, 'train')
-              for feat, path in paths.items()
-              if not feat.endswith('/extra')
-          },
-      ),
-      tfds.core.SplitGenerator(
-          name=tfds.Split.VALIDATION,
-          num_shards=calculate_num_shards(500, features_size_mb),
-          gen_kwargs={
-              feat:os.path.join(path, 'val')
-              for feat, path in paths.items()
-              if not feat.endswith('/extra')
-          },
-      ),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            num_shards=calculate_num_shards(2975, features_size_mb),
```

Comment:
You don't need to specify num_shards anymore.

Reply:
Wasn't aware of this - thanks!